backend/dataall/modules/metadata_forms/api/input_types.py

```diff
@@ -0,0 +1,33 @@
+from dataall.base.api import gql
+from dataall.base.api.constants import SortDirection
+from dataall.modules.metadata_forms.api.enums import EnvironmentSortField
+
+
+NewMetadataFormInput = gql.InputType(
+    name='NewMetadataFormInput',
+    arguments=[
+        gql.Field(name='name', type=gql.String),
+        gql.Field(name='description', type=gql.String),
+        gql.Field(name='SamlGroupName', type=gql.String),
+        gql.Field(name='visibility', type=gql.String),
+        gql.Field(name='homeEntity', type=gql.String),
```

Comment:
Should `name` , `SamlGroupName`, and `visibility` be NonNullable?

Reply:
[aea1745](https://github.com/data-dot-all/dataall/pull/1444/commits/aea1745ee8adcd027cb019f967e21c5c3a4849ca)